pkg/core/namespace.go

```diff
@@ -35,6 +35,12 @@ type Namespace struct {
 	Contracts   *MultipartyContracts `ffstruct:"Namespace" json:"-"`
 }
 
+type NamespaceWithInitStatus struct {
+	*Namespace
+	Initializing        bool   `ffstruct:"NamespaceWithInitStatus" json:"initializing,omitempty"`
```

Comment:
I'm trying to think if there is already precedence in the API for something like this. Intuitively I would have thought the field would have been `"ready"` or `"initialized"` and its default value would have been `false` until it was set to `true`. But I think I see why it's this way, because it's more consistent with the query parameter. Internally, elsewhere such as on the Namespace Manager we have `started = true/false`. I'm not sure if consistency there is something valuable or not. I'll leave it up to you.

Reply:
Yeah - was that reason, but also so that the new cruft of a boolean only turns up on the JSON in the case of something that is **not** ready.